kpatch-build/kpatch-elf.c

```diff
@@ -76,6 +76,19 @@ int is_debug_section(struct section *sec)
 	       !strncmp(name, ".eh_frame", 9);
 }
 
+/*
+ * ppc64le ABI V2 has different function prologues for
+ * gcc5 and gcc6, this check is required for ppc64le
+ * only.
+ */
+int gcc6_check(void)
+{
+#ifdef __powerpc__
+	return (GCC_VERSION >= 60100);
+#endif
+	return 0;
+}
```

Comment:
One potential problem with this GCC 6 check is that it checks the version which was used to compile create-diff-object itself, but that could have been a different version than was used to compile the kernel.  For example, kpatch-build could be delivered by the distro's RPM, but the user might be making a patch for another kernel.

However it's ok with me to leave this check for now and fix it later when somebody has a problem with it.